Disable running tests by default (we run them later in the build).
--- OpenBLAS-0.3.30.orig/Makefile
+++ OpenBLAS-0.3.30/Makefile
@@ -54,7 +54,7 @@
 .PHONY : all libs netlib $(RELA) test ctest shared install
 .NOTPARALLEL : shared
 
-all :: tests
+all :: shared
 	@echo
 	@echo " OpenBLAS build complete. ($(LIB_COMPONENTS))"
 	@echo
